### PR TITLE
Srr several fixes

### LINKED ISSUE #927: qudt:symol for unit:PPT is duplicate of unit:PPTH
The qudt:symbol for unit:PPT (part per trillion) ‰ is a duplicate of the qudt:symbol for unit:PPTH

it looks to be correct for unit:PPTH
https://en.wiktionary.org/wiki/%E2%80%B0

so qudt:symbol is likely incorrect for unit:PPT

I do not know what qudt:symbol for unit:PPT should be changed to so haven't raised a pull request.

### PR DESCRIPTION
**Pull request by steveraysteveray**
Resolves #927  and #928 .

### CODE REVIEW

**Comment by steveraysteveray** [comment]
Also resolves #940

**Review by fkleedorfer — APPROVED** [review]

File: vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl [hunk 1]
```diff
@@ -38480,10 +38483,11 @@ unit:PPT
   qudt:applicableSystem sou:SI ;
   qudt:conversionMultiplier 0.000000000001 ;
   qudt:conversionMultiplierSN 1.0E-12 ;
+  qudt:exactMatch unit:PPTR ;
   qudt:hasDimensionVector qkdv:A0E0L0I0M0H0T0D1 ;
   qudt:hasQuantityKind quantitykind:DimensionlessRatio ;
   qudt:iec61360Code "0112/2///62720#UAD927" ;
-  qudt:symbol "‰" ;
+  qudt:symbol "ppt" ;
```
Comment (ralphtq) [inline vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:38490]: why this change?

**Review by ralphtq — APPROVED** [review]

File: vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl (hunk 1 above)
Comment (steveraysteveray) [reply vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:38490]: Because the symbol that had been there means "part per thousand" which we have as unit:PPTH. 
unit:PPT means "part per trillion".
